starky/Cargo.toml

```diff
@@ -23,6 +23,7 @@ anyhow = { workspace = true }
 hashbrown = { workspace = true }
 itertools = { workspace = true }
 log = { workspace = true }
+serde = { workspace = true, features = ["rc"] }
```

Comment:
This is the same import in `plonky2`, it'd be best to import it at the workspace level and then

`serde = { workspace = true }`